lib/travis/model/request/states.rb

```diff
@@ -53,6 +53,9 @@ def finish
     end
 
     def add_build
+      if config && config[:language]
+        config[:language] = config[:language].to_s.downcase
```

Comment:
@BanzaiMan:  Ok, so after some problems with the deploys on staging, I tested this, and it is working correctly.

I don't think this is supposed to be a symbol... I just want to downcase its value to pass it to the build config, and leave the value as it is on the config file.